types/role-helpers.d.ts

```diff
@@ -1,9 +1,19 @@
-export function logRoles(container: HTMLElement): string
+export function logRoles(
+  container: HTMLElement,
+  options?: LogRolesOptions,
+): string
+
+type LogRolesOptions = {
```

Comment:
I think we use `interface`s in other locations.
Or is there a reason why this is a `type`?

```suggestion
interface LogRolesOptions {
```

Reply:
The reason was that I couldn't think of a reason to allow interface inheritance with that type, but it's not super crucial.
Will change it tomorrow.